www/assessment/reset-response.php

```diff
@@ -0,0 +1,209 @@
+    'rendering_type' => 'inline',
+    'type' => 'local_practice',
+    'session_id' => Uuid::generate(),
+    'user_id' => '$ANONYMIZED_USER_ID',
+    'organisation_id' => $roAdditionalOrgId,
+    'items' => [
+        [
+            'id' => 'demo_reset_response_act_1_no_edit',
+            'reference' => 'demo_reset_response_act_1_no_edit'
+        ]
+    ]
+];
+
+$InitDemo1 = new Init('items', $security, $consumer_secret, $requestDemo1);
+$signedRequestDemo1 = $InitDemo1->generate();
+
+/* Demo 2 configuration and signature sign */
+$requestDemo2 = [
+    'name' => 'Reset Response Demo 2',
+    'rendering_type' => 'inline',
+    'type' => 'local_practice',
+    'session_id' => Uuid::generate(),
+    'user_id' => '$ANONYMIZED_USER_ID',
+    'organisation_id' => $roAdditionalOrgId,
+    'items' => [
+        [
+            'id' => 'demo_reset_response_act_2_no_edit',
+            'reference' => 'demo_reset_response_act_2_no_edit'
+        ]
+    ]
+];
+
+$InitDemo2 = new Init('items', $security, $consumer_secret, $requestDemo2);
+$signedRequestDemo2 = $InitDemo2->generate();
+
+/* Demo 3 configuration and signature sign */
+$requestDemo3 = [
+    'name' => 'Reset Response Demo 3',
+    'rendering_type' => 'inline',
+    'type' => 'local_practice',
+    'session_id' => Uuid::generate(),
+    'user_id' => '$ANONYMIZED_USER_ID',
+    'organisation_id' => $roAdditionalOrgId,
+    'items' => [
+        [
+            'id' => 'demo_reset_response_act_3_no_edit',
+            'reference' => 'demo_reset_response_act_3_no_edit'
+        ]
+    ]
+];
+
+$InitDemo3 = new Init('items', $security, $consumer_secret, $requestDemo3);
+$signedRequestDemo3 = $InitDemo3->generate();
+
+?>
+    <div class="jumbotron section">
+        <div class="overview">
+            <h2>Reset Response</h2>
+            <br>
+            <p>Use the <a href="https://reference.learnosity.com/questions-api/methods#questionsApp-ResetResponse"><code>resetResponse</code></a> public method to allow user to reset the response of a question.</p>
+            <p>Below is a demo of the ‘resetResponse’(link to site reference) public method showing what can be achieved with a button that lets the user reset their response to a question. This can be useful in certain situations where end-users can be penalised for choosing an incorrect response. This public method provides a way for students to "reset" their response and leave a question unattempted if they’re not confident about the correct answer. This functionality is not available by default.</p>
+        </div>
+    </div>
+
+    <!--  Demo 1  -->
+    <div class="section pad-sml">
+        <h3">Multiple Choice</h3>
+        <hr>
+        <span class="learnosity-item" data-reference="demo_reset_response_act_1_no_edit"></span>
+        <div class="button-wrapper">
+            <button
+                type="button"
+                class="resetResponseButton"
+                onclick=resetResponse(1)
```

Comment:
Even in a Demo, we should always AVOID hardcoding `onclick` it's a bad practice.
Instead, I would recommend using data-attribute to refer to the questions App you want to refer to. In the script then use `addEventListener` to add `onclick` event instead